katello/katello-backup

```diff
@@ -1,94 +1,113 @@
 #!/bin/bash
 
 usage(){
-  echo "Usage: katello-backup <backup directory>"
-  echo " eg: $ katello-backup /tmp/katello-backup/"
+  echo "Usage: katello-backup --directory <backup directory>"
+  echo " eg: $ katello-backup --directory /tmp/katello-backup/"
```

Comment:
I felt like it was nicer in case down the line we wanted to add other arguments to the script. Open for discussion however.